source/blog/2018-06-08-the-emberjs-times-issue-50.md

```diff
@@ -13,14 +13,15 @@ Read either on the [Ember blog](https://www.emberjs.com/blog/2018/06/08/the-embe
 
 ---
 
-## [SECTION TITLE](#section-url)
-
-
+## [RFCS Deprecating Ember Utils](https://github.com/emberjs/rfcs/pull/334)
```

Comment:
Is `RFCS` referring to `RFC`s or is it another term I don't know of yet? If not, would just `RFC` (=Request for Comments) be suitable?

Reply:
RFCS is a brain fart 🙈